### ISSUE TITLE
[Feature request] Bind keys to run a script

### ISSUE THREAD

**Issue by rien333** [panzi/qjoypad#28]
Assigning the buttons to a script (maybe with the pressed button as an argument) would open up so much power, and should be fairly simple to do (something with a `system` call). The user could specify a path to a script in a text box somewhere.

Not sure if this in scope.

**Comment by panzi** [comment]
Not really in scope for me. Since the original maintainers seem to have abandoned this project I just took on the task of maintaining it so it will keep working. Not planning on adding any more features. I already did add the feature of live gamepad detection (instead of only scanning for gamepads on startup), but that's about it. Sorry.

However, if you write such a feature in a clean way that has no shell injection vulnerabilities and such and doesn't mess up the source too much I'd accept a pull request. However, there are several(!) other similar programs as this that are more actively maintained. Who knows, maybe they even support such a feature already. E.g.: https://github.com/AntiMicro/antimicro

Anyway, it is small and simple program at the moment, which makes the task of keeping it running with new Qt and Linux versions non-daunting. If it would get too complex with that new feature my motivation of keeping it maintained would lower. ;)
